src/main/java/uk/gov/justice/hmpps/prison/api/resource/SmokeTestHelperResource.kt

```diff
@@ -42,16 +42,14 @@ class SmokeTestHelperResource(private val service: SmokeTestHelperService) {
       content = [Content(mediaType = "application/json", schema = Schema(implementation = ErrorResponse::class))],
     ),
   )
-  @Operation(summary = "Sets imprisonment status smoke test data for this offender")
-  @PostMapping("/offenders/{offenderNo}/imprisonment-status")
+  @Operation(summary = "Sets the status for this offender to IN if it currently OUT, with smoke test data")
+  @PutMapping("/offenders/{offenderNo}/status")
   @ProxyUser
-  fun imprisonmentDataSetup(
```

Comment:
Old imprisonmentDataSetup is no longer required (was used for probation but no longer used)